metabrainz/templates/users/signup-commercial.html

```diff
@@ -43,15 +43,15 @@ <h1 class="page-title">Sign up <small>Commercial</small></h1>
       <div class="col-sm-5">{{ form.org_name(class="form-control", required="required") }}</div>
       <div class="col-sm-offset-4 col-sm-5" style="padding-top: 7px">
         <em class="text-muted">
-          If you don't have an organization name, you probably want to sign up as a
-          <a href="{{ url_for('users.signup_noncommercial') }}">non-commercial / personal</a> user.
+          {{ _('If you don't have an organization name, you probably want to sign up as a
+          <a href="{{ url_for('users.signup_noncommercial') }}">non-commercial / personal</a> user.') }}
         </em>
       </div>
     </div>
 
     <div class="form-group">
       <label class="col-sm-4 control-label">
-        MusicBrainz Account
+        MusicBrainz {{ _('Account') }}
```

Comment:
Why is `MusicBrainz` not part of this one? Ordering might be different in some languages.